WordPress/src/main/res/layout/scan_fragment.xml

```diff
@@ -1,5 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" />
+    android:layout_height="match_parent"
+    android:padding="@dimen/margin_extra_large">
+
+    <org.wordpress.android.ui.prefs.EmptyViewRecyclerView
+        android:id="@+id/recycler_view"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
```

Comment:
Question (❓ ): I think I asked @malinajirka about this and I got the understanding that in this codebase the preferred way is to do `layoutManager` programmatically instead. Please let me know your thoughts?

PS: I only ask these questions, actually all my questions so far, because when I start contributing as well, I want to be sure I using the right and agreed approach. Although, I too believe that having it here might be better. 😊

Reply:
No preference as such, it could be set directly in the xml so I did that. If it had been a custom layout manager then of course it would need to be added in the code.

Reply:
Definitely, this is my exact thinking as well! 🌟

It just that I had a similar discussion with Jirka (if I remember correctly) but the other around, thus I pinged him. 😊

Reply:
👍 yeah, I usually set the manager in code and I think that's what most devs are doing. Having said that it's such a minor thing that I guess it doesn't matter and we can leave it on personal preference.